kitsune/notifications/api.py

```diff
@@ -31,7 +31,7 @@ def has_object_permission(self, request, view, obj):
 
 class NotificationSerializer(serializers.ModelSerializer):
     is_read = serializers.ReadOnlyField()
-    timestamp = DateTimeUTCField(source='action.timestamp')
+    timestamp = DateTimeUTCField('%Y-%m-%dT%H:%M:%SZ', source='action.timestamp')
```

Comment:
Why was this change necessary? Is this a fix from the previous upgrade of DRF?

Reply:
I believe this is coming from the replacement of `MySQL-Python` with `mysqlclient`. Probably the former did not return any microseconds.

Reply:
Oh right. Interesting. Thanks.